Cm/update for c14 (#3)
AdaGold/js-calculator: README.md

We will build a calculator command line interface (CLI) to perform simple arithmetic. The program should use an input variable that contains two numbers and an operation to provide the result of applying the operation to the two numbers.

Our JavaScript projects, **our projects will NOT take user input,** and instead **we will use a variable to hold and represent user input.**

## Setup Requirements

1. Fork this project repo
1. Clone this repo onto your computer
1. Run `npm install` to install dependencies
1. Work on this project, mostly within `calculator.js`

## Getting Started

> You should write your program as a series of statements within the file `calculator.js`. Anyone with this file you should be able to run the command `node calculator.js` to execute the program.

Reviewer: "anyone with the file you", not sure which direction you were going with this so I just made it parenthetical.

```suggestion
You should write your program as a series of statements within the file `calculator.js`. Anyone with this file (including you) should be able to run the command `node calculator.js` to execute the program.
```

</details>

## Manual Testing

> We haven't dived into unit tests in JavaScript yet, so you may want to try to test your function manually.  You can run the function and use terminal-based user input with `npm start`, or you can run `calculator.js` manually with `node calculator.js`.  There is an example of manually testing the function in `calculator.js` as a comment.

Reviewer: ```suggestion
We haven't dived into unit tests in JavaScript yet, so _you must test your function manually._  You can run the function and use terminal-based user input with `npm start`, or you can run `calculator.js` manually with `node calculator.js`.  There is an example of manually testing the function in `calculator.js`, written as a comment.
```
